packages/instrument-utils/src/guards/shouldUseStripeLinkByMinimumAmount.ts

```diff
@@ -0,0 +1,47 @@
+import { Cart } from '@bigcommerce/checkout-sdk';
+
+enum stripeLinkMinAmount {
```

Comment:
Could you provide the basis for setting the value？

Reply:
@bc-peng it was taken from the Stripe docs. I'll add a link for a Stripe documentation